src/scss/_breadcrumb.scss

```diff
@@ -56,23 +58,27 @@
 .jazz-breadcrumb > ol > li span {
   @include jazz-theme-color(breadcrumb-item);
   text-decoration: none;
-  //padding: px-to-rem(5) 0 px-to-rem(5) px-to-rem(5);
   padding: px-to-rem(5);
+  border-bottom: px-to-rem(2) solid transparent;
 }
 
 .jazz-breadcrumb > ol > li a:hover {
   @include jazz-theme-color(breadcrumb-hover);
-  border-bottom: px-to-rem(1) solid jazz-theme-rgb(breadcrumb-hover-line);
+  border-bottom: px-to-rem(2) solid jazz-theme-rgb(breadcrumb-hover-line);
 }
 
 .jazz-breadcrumb > ol > li a:active,
 .jazz-breadcrumb > ol > li a[aria-current=page],
 .jazz-breadcrumb > ol > li span[aria-current=page]{
   @include jazz-theme-color(breadcrumb-active);
   font-weight: 600;
-  border-bottom: px-to-rem(1) solid jazz-theme-rgb(breadcrumb-hover-line);
+  border-bottom: px-to-rem(2) solid jazz-theme-rgb(breadcrumb-hover-line);
 }
 
 .jazz-breadcrumb--flush-left {
   margin-left: -10px;
 }
+
+//.jazz-breadcrumb-spacer {
```

Comment:
Now that you have removed this, you can remove the comment.  I tend to do this also, but we will want to make sure that the code stays clean.  Keep in mind that as long as you already have the code check into a Git version, you can always get back to it through the Git history.